test/helpers/mockConfig.ts

```diff
@@ -36,6 +36,8 @@ const mockConfig = (preConfig?: any) => {
     walletStores: wallet,
     walletStoresAsPrivateKeys: true
   });
+  config.client = client;
```

Comment:
This also shouldn't be necessary.

Reply:
Part of the mock implementation for tests